chia/README.md

```diff
@@ -48,7 +48,8 @@ version: "2.0"
 
 services:
   chia:
-    image: cryptoandcoffee/akash-chia:72
+    image: cryptoandcoffee/akash-chia:latest
+    imagePullPolicy: Always
```

Comment:
you have to use version numbers.  we don't pass through `imagePullPolicy`.

Reply:
OK! I will remove the `imagePullPolicy: Always`

In regards to version numbers, I have updated helm charts to make sure the provider has `imagePullPolicy`  set now to `Always`, I wasn't 100% sure if we needed it in the SDL.  Now that I removed that based on your suggestion, I re-tested on the providers with `imagePullPolicy : Always` and the behavior is as expected with akash-chia:latest, where if an update exists it will retrieve it.  I would like to keep the akash-chia:latest for now as we send deployments to providers with `imagePullPolicy : Always` and additionally will open PR on the helm-charts.

Reply:
We always set `PullIfNotPresent`.  See [here](https://github.com/ovrclk/akash/blob/3b48bf812c8d3da3e53d5a10c81613316ab3a965/provider/cluster/kube/builder/workload.go#L56).

Reply:
> I have updated helm charts to make sure the provider has imagePullPolicy set now to Always

What update did you make?

Reply:
OK, I understand better now with your link to akash codebase, the imagePullPolicy is not controlled with the helm charts.  I was trying to adapt https://github.com/ovrclk/helm-charts/blob/main/charts/akash-provider/values.yaml to use imagePullPolicy.  Thank you for this.  I will revert back to version number 72 - and push update under that.